recipe/recipe.yaml

```diff
@@ -0,0 +1,135 @@
+    patches:
+      # test that fails due to timing issues on slow CI agents when in emulation
+      - patches/0001-loosen-time-limit-in-test_scipy_mi_time_limit_reache.patch
+
+build:
+  number: 1
+
+outputs:
+  - package:
+      name: cvxpy-base
+      version: ${{ version }}
+    build:
+      script:
+        # if not in release mode, this may lead to issues, see
+        # https://github.com/cvxpy/cvxpy/issues/1690
+        - if: unix
+          then:
+            - export CXXFLAGS="${CXXFLAGS} -DNDEBUG"
+            - rm $SRC_DIR/pyproject.toml
+          else:
+            - del %SRC_DIR%\pyproject.toml
+        - python -m pip install . --no-deps -vvv
+    requirements:
+      build:
+        - if: build_platform != target_platform
+          then:
+            - python
+            - cross-python_${{ target_platform }}
+            - numpy
+        - ${{ compiler('c') }}
+        - ${{ stdlib("c") }}
+        - ${{ compiler('cxx') }}
+      host:
+        - python
+        - pybind11
+        # conda-forge recipe inserts oldest numpy per platform/runtime
+        - numpy
+        - pip
+        - setuptools
+      run:
+        - python
+        - scipy >=1.1.0
+    tests:
+      - python:
+          imports:
+            # public interface is defined (see #75) to be the content of
+            # https://github.com/cvxpy/cvxpy/blob/master/cvxpy/__init__.py
+            - cvxpy
+
+  - package:
+      name: cvxpy
+      version: ${{ version }}
+    requirements:
+      host:
+        - python
+      run:
+        - python
+        - ${{ pin_subpackage('cvxpy-base', exact=True) }}
+        - osqp >=0.6.2
+        - clarabel >=0.5
+        - cffi
+        - scs >=3.2.4
+    tests:
+      - python:
+          imports:
+            - cvxpy
+            - cvxpy.cvxcore
+            - cvxpy.cvxcore.python
+      - script:
+          - if: unix
+            then:
+              - |
+                tests_to_skip="_not_a_real_test"
+                # accuracy failure on PPC, likely due to emulation
+                if [ "${{ target_platform }}" == "linux-ppc64le" ]; then
+                  tests_to_skip="${tests_to_skip} or (TestDqcp and test_basic_multiply)"
+                fi
+                # test that fails on Python 3.13
+                python_version=$(python -c "import sys; print(f'{sys.version_info.major}.{sys.version_info.minor}')")
+                if [ "$python_version" == "3.13" ]; then
+                  tests_to_skip="${tests_to_skip} or test_sparsity_condition"
+                fi
+                pytest cvxpy/tests -v -k "not (${tests_to_skip})"
+              # test that pip recognizes installation correctly, see https://github.com/cvxpy/cvxpy/issues/2389;
+              # 'grep -v' inverts the match but returns non-zero exit code if no lines are returned; so we use 'wc -l';
+              - if [ 0 -eq $(pip list | grep "UNKNOWN" | wc -l) ]; then exit 0; else (pip list && exit 1); fi
```

Comment:
Also, this test is better expressed as a `files:` test, see also [torchaudio](https://github.com/conda-forge/torchaudio-feedstock/blob/28547e766acef0cfb4228626c89b27e9021dfe06/recipe/recipe.yaml#L151-L154)